packages/builder/src/components/automation/SetupPanel/AutomationBlockSetup.svelte

```diff
@@ -140,6 +149,31 @@
       ? [hbAutocomplete([...bindingsToCompletions(bindings, codeMode)])]
       : []
 
+  $: fetch = tableId
+    ? fetchData({
+        API,
+        datasource: {
+          type: "table",
+          tableId,
+        },
+        options: {
+          sortColumn: primaryDisplay,
+          sortOrder: SortOrder.ASCENDING,
+          query: {
+            fuzzy: {
+              [primaryDisplay]: rowSearchTerm || "",
+            },
+          },
+          limit: 20,
+        },
+      })
+    : null
+  $: fetchedRows = $fetch?.rows
+  $: fetchLoading = $fetch?.loading
+  $: primaryDisplay = table?.primaryDisplay
+  $: rowSearchTerm = ""
+  $: selectedRow = null
```

Comment:
If `rowSearchTerm` and `selectedRow` are simply normal variables that you're declaring here, it would be better to just declare them with `let` instead of a reactive statement. Functionally this will be the same, but a reactive statement like this is ambiguous as to whether it's declaring a new variable or assigning a value to an existing one (if you had previously declared it above). So for readability, I always prefer to only use reactive statements for logic that we actually want to be reactive. Declaring a variable via either a reactive statement or via `let` etc has no impact on whether other code can "react" to changes of those variables.